### ISSUE TITLE
server.httpServer is null

### ISSUE THREAD

**Issue by jamesarosen** [matthewp/astro-fastify#19]
When I follow the instructions for adding `astro-fastify` to an existing (minimal) Astro app, then run `pnpm run build`, I get this error:

```
error   Cannot read properties of null (reading 'closeAllConnections')
File:
  /path/to/node_modules/.pnpm/fastify@4.12.0/node_modules/fastify/fastify.js:192:54
```
I traced that to `server.httpServer` being `null` in [`serverFactory`](https://github.com/matthewp/astro-fastify/blob/b470dbb920c135af3df4c4ca522eae49fb24c2e0/lib/index.js#L42).

Is there configuration I'm missing?

**Comment by jcdogo** [comment]
+1 Same issue here

**Comment by smnbbrv** [comment]
actually `astro check` fails with the very same error

**Comment by matthewp** [comment]
Fixed by #20
